core/src/main/java/io/questdb/cutlass/line/AbstractLineTcpSender.java

```diff
@@ -59,23 +59,14 @@ public AbstractLineTcpSender(LineChannel channel, int bufferCapacity, int maxNam
     @Override
     public final void at(long timestamp, ChronoUnit unit) {
         putAsciiInternal(' ');
-        try {
-            put(NanosTimestampDriver.INSTANCE.from(timestamp, unit)).putAsciiInternal('n');
-        } catch (ArithmeticException e) {
-            put(MicrosTimestampDriver.INSTANCE.from(timestamp, unit)).putAsciiInternal('t');
-        }
-
+        putTimestamp(timestamp, unit);
```

Comment:
If I understand this correctly, it sends the designated timestamp at the end of the line. This timestamp token should not have any `t`, `n` suffixes, but the old code and the new code add the suffix. This is wrong; the designated timestamp should always use nanos without any suffix.

Reply:
Agreed that we keep this Influxdb compatibility for V1 only.
V2 has already diverged.